appveyor.yml

```diff
@@ -0,0 +1,32 @@
+version: "{build}"
+
+clone_depth: 10
+
+branches:
+  only:
+    - master
+
+build: off
+
+install:
+  - gem update --system
+  - gem install bundler --version 1.16.2
```

Comment:
Is this really needed?

Reply:
Nope. But `bundler-1.16.3` is known to cause build failures on Travis CI..

Reply:
Does AppVeyor use 1.16.3? I mean, it's pretty new anyway.

Reply:
Oh, I never checked that.. I'll check and make the change. Thanks :+1: